metsrw/premis/event.py

```diff
@@ -0,0 +1,154 @@
+from .. import exceptions
+from .. import utils
+
+
+class Event(object):
+    """
+    An object representing a PREMIS object element.
+
+    :param str event_type: Event type.
+    :param str outcome: Whether an event passed or failed.
+    :raises exceptions.ParseError: If the root is not event.
+    """
+
+    def __init__(self, identifier_type=None, identifier_value=None, event_type=None, event_datetime=None, detail=None, outcomes=None, linking_agent_identifiers=[], linking_object_identifiers=[]):
+        if identifier_type is None:
+            raise exceptions.ConstructError('identify_type argument is required.')
+
+        if identifier_value is None:
+            raise exceptions.ConstructError('identify_value argument is required.')
+
+        if event_type is None:
+            raise exceptions.ConstructError('event_type argument is required.')
+
+        if event_datetime is None:
+            raise exceptions.ConstructError('event_datetime argument is required.')
```

Comment:
The easier way to do this is to not provide a default value in the constructor.  ege `def __init__(self, identifier_type, identifier_value, event_type, event_datetime, detail=None, ...)`